testsuite/gnat2goto/tests/arrays_access/arrays_access.adb

```diff
@@ -0,0 +1,7 @@
+procedure Arrays_Access is
+   type Arr7 is array (1 .. 3) of Integer;
+   Actual7 : Arr7 := (1,2,3);
```

Comment:
Is this the bit that's triggering your problems with `array_copy` on the assign/read task? What if you leave it off, i.e. leave `Actual7` uninitialized like this: `Actual7 : Arr7;`?

Reply:
It helped a bit, ta.